config-connector/solutions/iam/helm/folder-iam/README.md

```diff
@@ -0,0 +1,83 @@
+# Folder IAM
+
+==================================================
+
+## NAME
+
+  folder-iam
+
+## SYNOPSIS
+
+  Config Connector compatible YAML files to grant a specific member a role (default to `roles/resourcemanager.folderEditor`) to an existing folder.
+
+## CONSUMPTION
+
+  1. Clone GoogleCloudPlatform/cloud-foundation-toolkit repository
+  
+      ```bash
+      git clone https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git
+      ```
+
+  1. Go to the folder-iam folder:
+
+      ```bash
+      cd cloud-foundation-toolkit/config-connector/solutions/iam/helm/folder-iam
+      ```
+
+## REQUIREMENTS
+
+1. GKE Cluster with Config Connector and [Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable_workload_identity_on_a_new_cluster).
+1. [Helm](../../../README.md#helm)
```

Comment:
In addition to the requirements you've already mentioned, the cnrm-system service account needs to have roles/resourcemanager.folderIamAdmin role in the desired folder.

Reply:
Added. Thanks!